Xamarin.PropertyEditing/ViewModels/PanelViewModel.cs

```diff
@@ -246,19 +271,20 @@ protected override void OnAddEditors (IEnumerable<EditorViewModel> editors)
 					}
 				}
 
-				string key = grouping.Key ?? String.Empty;
-				if (remainingItems != null) // TODO: pretty sure this was out of order before, add test
-					this.arranged.Add (key, new PanelGroupViewModel (key, grouping.Where (evm => remainingItems.Contains (evm))));
-				else
-					this.arranged.Add (key, new PanelGroupViewModel (key, grouping, separateUncommon: !isFlat));
+				string key = grouping.Key;
+				if (remainingItems != null) {// TODO: pretty sure this was out of order before, add test
+					if (remainingItems.Count > 0)
+						this.arranged.Add (key, new PanelGroupViewModel (TargetPlatform, key, grouping.Where (evm => remainingItems.Contains (evm))));
+				} else
+					this.arranged.Add (key, new PanelGroupViewModel (TargetPlatform, key, grouping, separateUncommon: !isFlat));
```

Comment:
Not from this change, but it's unclear why we care about `separateUncommon` in one case and not the other. Might want to add a comment when you get the chance.

Reply:
In the first case we can assume we're in a grouped mode as there wouldn't be remaining items otherwise. We could pass the same `!isFlat` to make things clearer I suppose, it would always be `true`.